Mage/src/main/java/mage/cards/ExpansionSet.java

```diff
@@ -376,7 +374,7 @@ public boolean hasBasicLands() {
         return null;
     }
 
-    public boolean isCustomSet() { return isCustomSet; }
+    public boolean isCustomSet() { return setType == SetType.CUSTOM_SET; }
```

Comment:
Please format the if blocks or single statement always with new lines as block. (should be adjustable in the IDE) 
Like:
```
public boolean isCustomSet() { 
    return setType == SetType.CUSTOM_SET; 
}
```